src/minimap.cpp

```diff
@@ -103,7 +103,7 @@ void Minimap::UpdateButtons()
  */
 Point32 Minimap::GetRenderingBase(const Rectangle32 &widget_pos) const
 {
-	const int required_size = this->zoom * (_world.GetXSize() + _world.GetYSize());
+	const unsigned required_size = this->zoom * (_world.GetXSize() + _world.GetYSize());
```

Comment:
This is opposite to using signed for sizes which you seemed to discuss in the last comment